doc/plugins/eu.esdihumboldt.hale.doc.user/html/tasks/data/save_transformation.html

```diff
@@ -31,11 +31,10 @@ <h1>Save the transformation result</h1>
 		work based on a XML target schema.
 	</p>
 	<p>
-		Supported data export formats in <i>hale�studio</i> are:
+		Supported data export formats in <i>hale�studio</i> can be viewed on the following page:
 	</p>
 	<ul>
-		<li><a href="../../reference/export/xml_data.html">XML</a></li>
-		<li><a href="../../reference/export/gml_data.html">GML</a></li>
+		<li><a href="../../reference/export/export.html">Supported formats for Export</a></li>
```

Comment:
Have we had a chance now to verify if that is working as expected? The discussion [here](https://github.com/halestudio/hale/pull/1093#discussion_r1447089800) does not seem to have come to a clear result if I understand it correctly.

Reply:
I tested it locally and it works as expected.